src/components/SpaceProposalPage.vue

```diff
@@ -205,14 +177,11 @@ onMounted(() => {
           @clickVote="clickVote"
         />
         <SpaceProposalVotesList
-          :loaded="loadedVotes"
           :space="space"
           :proposal="proposal"
-          :votes="votes"
           :strategies="strategies"
           :user-vote="userVote"
-          :loading-more="loadingMore"
-          @loadVotes="loadMore(loadMoreVotes)"
+          @votes="votes = $event"
```

Comment:
Now when I refresh the page I can't see the votes block when votes after loading anymore. Probably better to use the composable instead of emitting

Reply:
https://www.loom.com/share/4584d6c74bec45cab21e3f562ad256b9

Reply:
Now we emit votes only to support prop for plugins